images/calico/configs/latest.cni.apko.yaml

```diff
@@ -0,0 +1,26 @@
+contents:
+  packages:
+    - calico-cni
+    - calico-cni-compat
+
+accounts:
+  groups:
+    - groupname: nonroot
+      gid: 65532
+  users:
+    - username: nonroot
+      uid: 65532
+      gid: 65532
+  run-as: 0
+
+entrypoint:
+  command: /opt/cni/bin/install
+
+environment:
+  PATH: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/cni/bin
```

Comment:
not blocking, just curious for other cni impls we’ll inevitably maintain, was this needed for something else calling the cnis without the full path?

Reply:
This wasn't needed to make the tests pass, but this is consistent with the upstream image is all.